### PR TITLE
Alfa patch01 way of the cross

### LINKED ISSUE #88: Add links to maps
As user
I want to have links to trips in map cards

**Requirements:**

- add link button to appropriate trip inside map cards (park, way of the cross, labirynth)
- add badge to map icons with trip links inside

### LINKED ISSUE #97: Add map of the way of the cross on trip
As a user
I want to see a Way of the Cross map

**Requirements:**

add map to the first step of the way of the cross trip

### LINKED ISSUE #99: Create kompas.sds.pl website
As a user
I want to be able to view website kompas.sds.pl

**Requirements:**

- create a website on WordPress
- add download app guide
- add feature descriptions
- add destination map
- add seminary's home website

### PR DESCRIPTION
**Pull request by darowsluk**
Fix #88 
Fix #97
Fix #99

### CODE REVIEW

File: lib/model/mapDatabase.dart [hunk 1]
```diff
@@ -181,10 +189,38 @@ MapData generateMapData() {
...
+      MapItem(
+        locationX: 1030,
+        locationY: 650,
+        uid: '06', // TODO: implement proper UID
+        imageAsset: "assets/images/trip01/trip01_01.jpg",
+        type: MapItemType.parking,
+        title: 'Parking',
+        subtitle: 'Serdecznie witamy',
+        description:
+            'Bardzo się cieszymy, że nas odwiedzasz i zapraszamy do zwiedzania naszego kompleksu klasztornego w Wyższym Seminarium Duchownym Salwatorianów w Bagnie.\n\nPrzygotowaliśmy dla Ciebie przewodnik w postaci niniejszej aplikacji, który zawiera kilka ciekawych podróży po naszym klasztornym terenie.\n\nJeśli jesteś gotowy rozpocząć z tego miejsca parkinkowego, to zapraszamy do gry terenowej dla całej rodziny!',
```
Comment (knutm92) [inline lib/model/mapDatabase.dart:215]: "parkinkowego" typo

File: lib/statics/colors.dart [hunk 2]
```diff
@@ -18,8 +18,8 @@ class AppColors extends Color {
   /// Tint 90%
   static const primaryWhite = Color(0xffeef1f1);
 
-  /// red[600]
-  static const accentRed = Color(0xffe53935);
+  ///
+  static const accentContrast = Color.fromARGB(255, 247, 149, 3);
```
Comment (knutm92) [inline lib/statics/colors.dart:22]: Just an observation- this color seems to fit better than any of the color styles proposed by Figma.

**Review by knutm92 — CHANGES_REQUESTED** [review]
The badges stand out on the map and the links are handy, nice!

Other than a small typo- the code looks good.

File: lib/model/mapDatabase.dart (hunk 1 above)
Comment (darowsluk) [reply lib/model/mapDatabase.dart]: Thanks. Fixed it.

**Review by knutm92 — APPROVED** [review]

**Pull request merged**